apps/researcher/src/messages/nl/messages.json

```diff
@@ -153,54 +153,54 @@
   "safety": "Veiligheid",
-  "safetyLabel": "De Veiligheidskennisgeving wordt gebruikt om aan te geven dat uiterste voorzichtigheid en zorg nodig is bij het bekijken en behandelen van dit item of deze collectie voor het welzijn en de gezondheid van individuen of gemeenschappen.",
+  "safetyLabel": "De _Veiligheids-notice_ wordt gebruikt om aan te geven dat uiterste voorzichtigheid en zorg nodig is bij het bekijken en behandelen van dit item of deze collectie, voor het welzijn en de gezondheid van individuen of gemeenschappen.",
   "viewing": "Bekijken",
-  "viewingLabel": "De Bekijkkennisgeving wordt gebruikt om aan te geven dat er beperkingen zijn op wie, hoe en wanneer een item/collectie kan worden bekeken, gedistribueerd en gecirculeerd. Dit omvat bestaande afbeeldingen of representaties van een bestaand of gedeaccesioneerd item binnen een collectie.",
+  "viewingLabel": "De _Bekijken-notice_ wordt gebruikt om aan te geven dat er beperkingen zijn op wie, hoe en wanneer een item/collectie kan worden bekeken, gedistribueerd en gecirculeerd. Dit omvat bestaande afbeeldingen of representaties van een bestaand of gedeaccesioneerd item binnen een collectie.",
   "withholding": "Achterhouden",
-  "withholdingLabel": "De Achterhouden Kennisgeving wordt gebruikt om aan te geven dat er nooit reproducties van dit item of deze collectie mogen worden gemaakt. Als er al bestaande foto's of reproducties zijn, mogen deze niet worden gebruikt in een tentoonstelling of publicatie in print, online of in enig ander formaat."
+  "withholdingLabel": "De _Achterhouden-notice_ wordt gebruikt om aan te geven dat er nooit reproducties van dit item of deze collectie mogen worden gemaakt. Als er al bestaande foto's of reproducties zijn, mogen deze niet worden gebruikt in een tentoonstelling of publicatie in print, online of in enig ander formaat."
 },
 "LocalContextsNoticesOverview": {
-  "title": "Local Context Notices",
-  "description": "De Local Context Notices ondersteunen de opname van lokale protocollen voor toegang en gebruik van cultureel erfgoed dat digitaal circuleert buiten gemeenschapscontexten. <link> Meer informatie over de Lokale Context kennisgevingen.</link>.",
+  "title": "Local Contexts Notices",
+  "description": "De Local Contexts Collections Care (CC) Notices zijn zichtbare identificatiekenmerken die speciaal ontwikkeld zijn voor instellingen en depots om goede culturele zorg en beheer van collecties te ondersteunen, geleid en gestuurd door inheemse protocollen en autoriteit. <link> Meer informatie over de Local Contexts Collections Care Notices.</link>.",
   "defaultProvider": "Colonial Collections Consortium",
-  "defaultLabel": "Dit is een standaardkennisgeving als er geen andere labels of kennisgevingen zijn toegewezen.",
+  "defaultLabel": "Dit is een standaard Notice in het geval er geen andere Notices zijn toegewezen.",
```

Comment:
Volgens mij hadden we vanochtend besloten om deze notice altijd te laten staan - dan klopt deze tekst niet meer.